SPECS-EXTENDED/pyflakes/pyflakes.spec

```diff
@@ -2,8 +2,8 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Name:           pyflakes
 # WARNING: When updating pyflakes, check not to break flake8!
```

Comment:
Might be worth building flake8 against this to verify everything works together.

Reply:
Great catch. Thanks

Reply:
The buddy Seems to be happy :) [PR-3816](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=241068&view=logs&j=75cba72d-09cb-503a-3927-4ab61ac4a179&t=246631ef-4e63-576d-3b21-bc52cd4b0a93&l=2103)